CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
+### Changed
+- [BREAKING] Harden types on ABI objects ([#6](https://github.com/MetaMask/metamask-eth-abis/pull/6))
```

Comment:
Can we explain how consumers ought to adapt to the breaking changes?

Reply:
Yes. This is going to be extremely rare, but if for some reason consumers have a function like this:

``` typescript
import { abiERC1155 } from "@metamask/metamask-eth-abis";

function doSomethingWithERC1155ABI(abi: typeof abiERC1155) {
  // ...
}
```

and attempt to call that function by passing in a custom-built ABI instead of reusing `abiERC1155`:

``` typescript
doSomethingWithERC1155ABI({
  inputs: [
    {
      internalType: 'whatever',
      name: 'my name is',
      type: 'i am a cool type'
    },
    {
      internalType: 'blah blah',
      name: 'hello hello',
      type: 'i am a very cool type'
    },
  ],
  stateMutability: 'who knows',
  type: 'yeah yeah',
});
```

that's not going to work at all. The only valid way to use this function after this change would be to literally use the ABI object:

``` typescript
doSomethingWithERC1155ABI(abiERC1155)
```

Reply:
So maybe we could say something like this?

> If you have a function that takes an argument with `typeof abiERC1155`, `typeof abiERC20`, or `typeof abiERC721`, that function must now literally take the corresponding ABI object.

Reply:
@mcmire do you want to push this change to the changelog?

Reply:
Ah yup! Sorry. Updated in `f8cf72e`.